### DISCUSSION TITLE
Hugo executable not available on Windows?

### DISCUSSION THREAD

**Discussion by Pietrrrek** [thuliteio/doks#91] [Q&A] [answered]
I've installed the theme accoring to the tutorial and ran `npm install` which also installs the extended version of Hugo, but after doing so trying to run e.g. `hugo new <...>` from PowerShell while being in the root of the project results in the following error message:

```
hugo : The term 'hugo' is not recognized as the name of a cmdlet, function, script file, or operable program. 
...
```

Installing Hugo through chocolatey solves the issue. Is this an issue on my part or is this expected behaviour?

**Comment by Pietrrrek** [comment]
I've tested it on multiple Windows machines, at first I thought that using an older version of Node might be the issue, but after installing the newest LTS version of Node and reinstalling the project from scratch nothing changed.

**Comment by h-enk** [comment, marked as answer]
Use the `npm run create` command, like so e.g. `npm run create -- blog/my-new-post.md`.

Note) When your run `npm run` you get a list of all available commands (or look in `package.json`). One of the commands is `create`.

P.S.) I will add this to/ update the documentation

**Comment by Pietrrrek** [comment]
Updating the documentation is perfect. 
Since I've got you here I would like to thank you for this template. It's really awesome and I've already started using it at work.
